### ISSUE TITLE
Rustdoc builds for `atsamx7x-hal` are failing

### ISSUE THREAD

**Issue by tgross35** [atsams-rs/atsamx7x-rust#47]
Currently there are no docs on docs.rs for the crate: https://docs.rs/crate/atsamx7x-hal/0.4.1

It seems like the build failed due to missing some feature flags

**Comment by tgross35** [comment]
It looks like this may be settable in Cargo.toml https://docs.rs/about/builds#cross-compiling

**Comment by tgross35** [comment]
The samd crate uses the following
```
[package.metadata.docs.rs]
features = ["samd21g", "samd21g-rt", "unproven", "usb"]
```

**Comment by tgross35** [comment]
Fix proposed in #49

**Comment by michalfita** [comment]
@tgross35 Good spot! Thank you. I somehow neglected the fact `docs.rs` are built for each published crate.

When #48 and #49 are in I release 0.4.2.
